index.html

```diff
@@ -0,0 +1,209 @@
+                        LightHouse Christian Outreach Center, Oregun, Ikeja, Lagos state.
+                    </p>
+                </div>
+                <div class="event-info">
+                    <h3 class="date">2022.09.25 (Fri) ~ 27(Sun)</h3>
+                    <p class="venue lato-font">
+                        @ LightHouse Christian Outreach Center
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!--Events Section-->
+    <section class="main-events">
+        <div class="main-events-container">
+            <h2 class="main-events-header">Community Day Events</h2>
+            <hr>
+            <div class="events-list">
+                <div class="event1 event">
+                    <img src="img/game1.png" alt="event-icon" class="event-img event-icon">
+                    <h2 class="lato-font">Games</h2>
+                    <p class="lato-font">
+                        Games including football, tennis, basketball, and board games.
+                    </p>
+                </div>
+
+                <div class="event2 event">
+                    <img src="img/game2.png" alt="event-icon" class="event-img event-icon">
+                    <h2 class="lato-font">Games</h2>
+                    <p class="lato-font">
+                        Games including football, tennis, basketball, and board games.
+                    </p>
+                </div>
+
+                <div class="event3 event">
+                    <img src="img/game3.png" alt="event-icon" class="event-img event-icon">
+                    <h2 class="lato-font">Games</h2>
+                    <p class="lato-font">
+                        Games including football, tennis, basketball, and board games.
+                    </p>
+                </div>
+
+                <div class="event4 event">
+                    <img src="img/game4.png" alt="event-icon" class="event-img event-icon">
+                    <h2 class="lato-font">Games</h2>
+                    <p class="lato-font">
+                        Games including football, tennis, basketball, and board games.
+                    </p>
+                </div>
+
+                <div class="event5 event">
+                    <img src="img/game5.png" alt="event-icon" class="event-img event-icon">
+                    <h2 class="lato-font">Games</h2>
+                    <p class="lato-font">
+                        Games including football, tennis, basketball, and board games.
+                    </p>
+                </div>
+            </div>
+
+            <div class="events-button-container">
+                <button class="events-button lato-font">
+                    Join Community Day 2022
+                </button>
+            </div>
+        </div>
+    </section>
+
+    <!--Featured Speakers Section-->
+    <div class="featured-speakers-heading">
+        <h2 class="lato-font">Featured Speakers</h2>
+        <hr>
+    </div>
+
```

Comment:
- [ ] [REQUIRED]  kindly check the image below:

|Your Design|
|----------|
|![image](https://user-images.githubusercontent.com/62938735/191991202-b21c12bd-7fde-4c61-9fb6-fa46d9208afe.png)|

- [ ] [REQUIRED] Kindly reduce the width and  make the speakers in the centre for the mobile version